UserDetailsClient/UserDetailsClient.iOS/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AsyncUsageAnalyzers" version="1.0.0-alpha003" targetFramework="xamarinios10" developmentDependency="true" />
-  <package id="Microsoft.Identity.Client" version="2.0.0-preview" targetFramework="xamarinios10" />
+  <package id="Microsoft.Identity.Client" version="2.0.1-preview" targetFramework="xamarinios10" />
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="xamarinios10" />
```

Comment:
><package id="Newtonsoft.Json" version="11.0.2" targetFramework="xamarinios10" /> [](start = 2, length = 80)

why?

Reply:
was testing the iOS simulator w/this sample and needed to add in the Newtonsoft library, but should have removed it before pushing this change. thanks.